src/DssSpell.t.sol

```diff
@@ -43,70 +43,109 @@ contract DssSpellTest is DssSpellTestBase {
+        assertEq(chainLog.getAddress("MCD_JOIN_WBTC_C"), addr.addr("MCD_JOIN_WBTC_C"));
+        assertEq(chainLog.getAddress("MCD_CLIP_WBTC_C"), addr.addr("MCD_CLIP_WBTC_C"));
+        assertEq(chainLog.getAddress("MCD_CLIP_CALC_WBTC_C"), addr.addr("MCD_CLIP_CALC_WBTC_C"));
 
-    }
+        assertEq(chainLog.getAddress("MCD_JOIN_PSM_GUSD_A"), addr.addr("MCD_JOIN_PSM_GUSD_A"));
+        assertEq(chainLog.getAddress("MCD_CLIP_PSM_GUSD_A"), addr.addr("MCD_CLIP_PSM_GUSD_A"));
+        assertEq(chainLog.getAddress("MCD_CLIP_CALC_PSM_GUSD_A"), addr.addr("MCD_CLIP_CALC_PSM_GUSD_A"));
+        assertEq(chainLog.getAddress("MCD_PSM_GUSD_A"), addr.addr("MCD_PSM_GUSD_A"));
 
-    function testAAVELerpOffboardings() public {
-        checkIlkLerpOffboarding("AAVE-A", "AAVE-A Offboarding", 165, 2100);
-    }
-    function testBALLerpOffboardings() public {
-        checkIlkLerpOffboarding("BAL-A", "BAL-A Offboarding", 165, 2300);
-    }
-    function testCOMPLerpOffboardings() public {
-        checkIlkLerpOffboarding("COMP-A", "COMP-A Offboarding", 165, 2000);
+        assertEq(chainLog.version(), "1.9.11");
     }
 
-    function testLRCIncreasedMatLerpOffboarding() public {
-        checkIlkLerpIncreaseMatOffboarding("LRC-A", "LRC Offboarding", "LRC-A Offboarding", 24300);
-    }
+    function testNewIlkRegistryValues() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
 
-    function testBATIncreasedMatLerpOffboarding() public {
-        checkIlkLerpIncreaseMatOffboarding("BAT-A", "BAT Offboarding", "BAT-A Offboarding", 11200);
+        // Insert new ilk registry values tests here
+        assertEq(reg.pos("WBTC-C"), 45);
+        assertEq(reg.join("WBTC-C"), addr.addr("MCD_JOIN_WBTC_C"));
+        assertEq(reg.gem("WBTC-C"), addr.addr("WBTC"));
+        assertEq(reg.dec("WBTC-C"), DSTokenAbstract(addr.addr("WBTC")).decimals());
+        assertEq(reg.class("WBTC-C"), 1);
+        assertEq(reg.pip("WBTC-C"), addr.addr("PIP_WBTC"));
+        assertEq(reg.xlip("WBTC-C"), addr.addr("MCD_CLIP_WBTC_C"));
+        assertEq(reg.name("WBTC-C"), "Wrapped BTC");
+        assertEq(reg.symbol("WBTC-C"), "WBTC");
+
+        assertEq(reg.pos("PSM-GUSD-A"), 46);
+        assertEq(reg.join("PSM-GUSD-A"), addr.addr("MCD_JOIN_PSM_GUSD_A"));
+        assertEq(reg.gem("PSM-GUSD-A"), addr.addr("GUSD"));
+        assertEq(reg.dec("PSM-GUSD-A"), DSTokenAbstract(addr.addr("GUSD")).decimals());
+        assertEq(reg.class("PSM-GUSD-A"), 1);
+        assertEq(reg.pip("PSM-GUSD-A"), addr.addr("PIP_GUSD"));
+        assertEq(reg.xlip("PSM-GUSD-A"), addr.addr("MCD_CLIP_PSM_GUSD_A"));
+        assertEq(reg.name("PSM-GUSD-A"), "Gemini dollar");
+        assertEq(reg.symbol("PSM-GUSD-A"), "GUSD");
     }
 
-    function testZRXIncreasedMatLerpOffboarding() public {
-        checkIlkLerpIncreaseMatOffboarding("ZRX-A", "ZRX Offboarding", "ZRX-A Offboarding", 5500);
-    }
+    function testDaiVests() public {
```

Comment:
I think we could move it in the base tests and add it to `testSpellIsCast_GENERAL` or we could follow a similar pattern as `checkCollateralValues` to keep track of the Dai Streamings with a general upper bond and local streams details as well (eg. `checkDaiStreams`)
cc: @gbalabasquer @brianmcmichael

Reply:
So I sort of did that with the recent commit. I added repeatable `checkDaiVest`. I think it should stay as a separate test.